tests/test_isrTask.py

```diff
@@ -477,9 +478,9 @@ def test_maskingCase_throughEdge(self):
         results = self.validateIsrResults()
 
         self.assertEqual(countMaskedPixels(results.exposure, "SAT"), 0)
-        self.assertEqual(countMaskedPixels(results.exposure, "INTRP"), 0)
-        self.assertEqual(countMaskedPixels(results.exposure, "SUSPECT"), 0)
-        self.assertEqual(countMaskedPixels(results.exposure, "BAD"), 0)
+        self.assertEqual(countMaskedPixels(results.exposure, "INTRP"), 40800)
+        self.assertEqual(countMaskedPixels(results.exposure, "SUSPECT"), 40800)
+        self.assertEqual(countMaskedPixels(results.exposure, "BAD"), 40800)
```

Comment:
Why do these values change so enormously?

Reply:
Fallout from the same bug (the suspect level was being set to zero).